kw_webapp/migrations/0003_auto_20160110_1715.py

```diff
@@ -11,13 +11,13 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.AddField(
+        migrations.AlterField(
```

Comment:
This had to be updated to `AlterField` because the migration directly prior adds this field.
https://github.com/Kaniwani/kw-backend/blob/master/kw_webapp/migrations/0002_auto_20160110_1624.py#L44